Frontend/ui-customer/src/Screens/CustomerUtils/ReportIssueChatScreen/View.purs

```diff
@@ -0,0 +1,484 @@
+            ]
+        , textView $ 
+            [ width MATCH_PARENT
+            , layoutGravity "center"
+            , color Color.black700
+            , visibility if state.data.showStillHaveIssue then GONE else VISIBLE
+            , text if state.data.isResolved then getString ISSUE_RESOLVED_TEXT else getString ISSUE_SUBMITTED_TEXT
+            , gravity CENTER
+            ] <> FontStyle.paragraphText TypoGraphy
+        ]
+    ]
+
+addAudioModel :: ReportIssueChatScreenState -> (Action -> Effect Unit) -> forall w. PrestoDOM (Effect Unit) w
+addAudioModel state push = AddAudioModel.view (push <<< AddAudioModelAction) (addAudioModelConfig state)
+
+addImageModel :: ReportIssueChatScreenState -> (Action -> Effect Unit) -> forall w. PrestoDOM (Effect Unit) w
+addImageModel state push = AddImagesModel.view (push <<< AddImagesModelAction) (addImageModelConfig state)
+
+recordAudioModel :: ReportIssueChatScreenState -> (Action -> Effect Unit) -> forall w. PrestoDOM (Effect Unit) w
+recordAudioModel state push = RecordAudioModel.view (push <<< RecordAudioModelAction) (recordAudioModelConfig state)
+
+viewImageModel :: ReportIssueChatScreenState -> (Action -> Effect Unit) -> forall w. PrestoDOM (Effect Unit) w
+viewImageModel state push = ViewImageModel.view (push <<< ViewImageModelAction) (viewImageModelConfig state)
+
+callModel :: ReportIssueChatScreenState -> (Action -> Effect Unit) -> forall w. PrestoDOM (Effect Unit) w
+callModel state push =
+  screenAnimationFadeInOut
+    $ linearLayout
+        [ width MATCH_PARENT
+        , height WRAP_CONTENT
+        , orientation VERTICAL
+        , onBackPressed push (const BackPressed)
+        , cornerRadius 16.0
+        , background Color.white900
+        , padding (Padding 16 24 16 24)
+        , margin (MarginHorizontal 16 16)
+        , gravity CENTER
+        ]
+        [ textView
+            [ text $ getString (if state.props.showCallDriverModel then CALL_DRIVER_TITLE else CALL_SUPPORT_TITLE)
+            , textSize FontSize.a_20
+            , fontStyle $ FontStyle.bold LanguageStyle
+            , color Color.black800
+            ]
+        , textView
+            [ text $ getString (if state.props.showCallDriverModel then CALL_DRIVER_DESCRIPTION else CALL_SUPPORT_DESCRIPTION "CALL_SUPPORT_DESCRIPTION")
+            , textSize FontSize.a_16
+            , margin (Margin 16 16 16 16)
+            , color Color.black800
+            , gravity CENTER
+            ]
+        , linearLayout
+            [ orientation VERTICAL
+            , height MATCH_PARENT
+            , width MATCH_PARENT
+            , margin (MarginTop 16)
+            , gravity CENTER
+            ]
+            [ linearLayout
+                [ weight 1.0
+                , margin (MarginBottom 4)
+                , height MATCH_PARENT
+                , width MATCH_PARENT
+                ]
+                [ PrimaryButton.view (push <<< ConfirmCall) (doneButtonConfig state) ]
+            , linearLayout
```

Comment:
Instead of writing it two times, Create a function and then use

`            , linearLayout
                [ weight 1.0
                , height MATCH_PARENT
                , width MATCH_PARENT
                ]
                [ PrimaryButton.view (push <<< CancelCall) (cancelButtonConfig state) ]`

Reply:
both are different button.

Reply:
@S-PRAVINKUMAR both are different button and what makes them different is the config and action so just pass those two things as a parameter to the function. which @Rohit4997 asked you to create.